src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -80,8 +82,8 @@ public void TimesIncludeMillisecondPart()
                 {
                     DateTime time = function.Getter(item);
                     msec = time.Millisecond;
-                    if (msec != 0)
-                        break;
+                    // if (msec != 0)
```

Comment:
To get info about the failing case you will either have to do a private Helix run, or actually commit this change.

If you want to commit it (which is simplest) then you may want to break out an unchanged copy of this test for Windows, so that it doesn't start failing too.

Reply:
Let me know if youw ant me to do this

Reply:
@danmosemsft If you would like to take this over, that would be awesome. Thanks!

My plan was to first make it fail with injected failure in CoreFX CI to see what kind of file system the CoreFX CI runs on. Then commit it and trigger corefx_baseline job in CoreCLR CI to see what kind of file system it reports there.

Reply:
OK!